src/docs/PCA/PCAUserGuide.adoc

```diff
@@ -35,23 +35,32 @@ include::references/HDFS-SPARK.adoc[]
 
 include::references/Storm.adoc[]
 
-=== Zookeeper
+=== Kafka
 
-image::PCA_zookeeper.png[align=center]
+The Kafka service allows to deploy through ProActive Cloud Automation (PCA) Portal a Kafka broker. 
 
-=== Kafka
+image::PCA_kafka.png[align=center][100000,2000]
 
-image::PCA_kafka.png[align=center]
+The instance will be in the state running after the deployment.
 
-=== Visdom
+image::PCA_kafka1.png[align=center][100000,2000]
+
+It can be finished after to release the ressources.
 
-image::PCA_visdom.png[align=center]
+image::PCA_kafka2.png[align=center][100000,2000]
 
-== HOW TO
+=== Zookeeper
+
+The Zookeeper instance lifecycle management is similar to the Kafka one. 
```

Comment:
This is wrong because Zookeeper doesn't depend on any other service unlike Kafka.